go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/broadinstitute/golang-project-template
+module github.com/broadinstitute/sherlock
 
 go 1.15
```

Comment:
el quicky, go version mismatch.

Reply:
el good catch 😄 I should probably remove go.mod from the template repo. That should really be generated per project.